packages/template-retail-react-app/app/hooks/use-password-reset.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {AuthHelpers, useAuthHelper} from '@salesforce/commerce-sdk-react'
+import {useToast} from '@salesforce/retail-react-app/app/hooks/use-toast'
+import {useIntl} from 'react-intl'
+import {useAppOrigin} from '@salesforce/retail-react-app/app/hooks/use-app-origin'
+import {getConfig} from '@salesforce/pwa-kit-runtime/utils/ssr-config'
+
+/**
+ * This hook provides commerce-react-sdk hooks to simplify the reset password flow.
+ */
+export const usePasswordReset = () => {
+    const showToast = useToast()
+    const {formatMessage} = useIntl()
+    const appOrigin = useAppOrigin()
+    const config = getConfig()
+    const resetPasswordCallback =
+        config.app.login?.resetPassword?.callbackURI || '/reset-password-callback'
```

Comment:
Likewise, in the `usePasswordReset` hook we read the callback path from the config file to send to the API call